src/test/java/org/hillel/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package org.hillel;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 

```

Comment:
You can write unit tests for your implementation or otherwise, I don't see any profit in this class and suggest removing it.